Add new class UBERON:8410000 duodeno-jejunal junction

paolaroncaglia opened the pull request:
Addresses #1657

paolaroncaglia commented:
@dosumis 
Could you please confirm if I used the 'Latin term' synonym property correctly? Thanks.

dosumis commented on uberon_edit.obo:
```diff
@@ -195227,6 +195227,27 @@ relationship: dc-creator http://orcid.org/0000-0003-2034-601X
+synonym: "duodeno-jejunal flexure" EXACT []
+synonym: "duodenojejunal flexure" EXACT []
+synonym: "duodenojejunal junction" EXACT []
+synonym: "flexura duodenojejunalis" EXACT [] {http://purl.obolibrary.org/obo/uberon/core#LATIN=""}
```
I think EXACT is fine, but that LATIN should be a synonym_type.  Did you check other examples in Protege?

paolaroncaglia replied:
@dosumis 

> I think EXACT is fine, but that LATIN should be a synonym_type. Did you check other examples in Protege?

I see two annotation properties, "LATIN" and "latin term" (only the latter is a subclass of synonym_type_property), and there are no annotation property usage examples for either (except for my term duodeno-jejunal junction). That's weird as I _think_ I've seen at least one of those properties used. I'll change the AP to "latin term" as a synonym_type_property, and I'll add to the agenda for the next CL/Uberon meeting.

dosumis replied:
Yep - should be 'latin term' in Protege see
![image](https://user-images.githubusercontent.com/112839/104635568-f7a98880-5699-11eb-8aef-6def83e58595.png) - but this will show up as LATIN in OBO.

dosumis replied:
There probably needs to be some clean up of cl-edit.owl to remove incorrect term - but we need to be careful:  Protege bug means usage is not shown.

paolaroncaglia replied:
> There probably needs to be some clean up of cl-edit.owl to remove incorrect term - but we need to be careful: Protege bug means usage is not shown.

As discussed at the CL editors meeting today, this will be addressed in https://github.com/obophenotype/uberon/issues/1727. Note, the duplication is in Uberon but not CL, so no action required there.

dosumis reviewed (changes requested):
pubs in xrefs -> def xrefs

paolaroncaglia commented:
@dosumis 
I addressed your comments, including the latin term, even though that's not showing up as a diff. Here's what it looks like in Protege for me:
![Screen Shot 2021-01-14 at 11 28 20](https://user-images.githubusercontent.com/4982516/104578976-aaa6c180-565b-11eb-9d87-82e3b3c5ffed.png)
I changed the Wikipedia version to what you suggested in https://github.com/obophenotype/uberon/pull/1718. It doesn't resolve as a live link, but I hope it's fine? There are no other examples of the same in Protege. Thanks.
